lib/runner.js

```diff
@@ -69,6 +71,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		opts.serial = true;
 	}
 
+	if (opts.type === 'test' && title !== null && (this._match && this._match.length > 0)) {
```

Comment:
Nope, not needed. This would be a case of me not killing my darlings. The amount of times I've been recommended to go and check out lisp is bordering on the ridiculous.